src/adaptermanager.js

```diff
@@ -82,7 +85,41 @@ exports.aliasBidAdapter = function (bidderCode, alias) {
   }
 };
 
+exports.registerAnalyticsAdapter = function ({ adapter, code }) {
+  if (adapter && code) {
+
+    if (typeof adapter.enable === CONSTANTS.objectType_function) {
+      adapter.code = code;
+      _analyticsRegistry[code] = adapter;
+    } else {
+      utils.logError(`Prebid Error: Analytics adaptor error for analytics "${code}"
+        analytics adapter must implement an enableAnalytics() function`);
+    }
+  } else {
+    utils.logError('Prebid Error: analyticsAdapter or analyticsCode not specified');
+  }
+};
+
+exports.enableAnalytics = function (config) {
+  if (!utils.isArray(config)) {
+    config = [config];
+  }
+
+  utils._each(config, adapterConfig => {
```

Comment:
I added code to replace `enableAnalytics` with a warning here:
https://github.com/prebid/Prebid.js/pull/297/files#diff-84954ceef9a26dfb9a93dfafbd10c8e4R69